google/cloud/universe_domain/demo/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Google LLC
```

Comment:
AFAICT, we have no cmake-based builds building this demo. Is that intentional?

I think we need at least one, even if the demo is temporary.